packages/ibm-products/src/components/InlineTip/InlineTip.js

```diff
@@ -0,0 +1,269 @@
+              <div className={`${blockClass}__secondary-btn`}>{contentCTA}</div>
+            )}
+          </section>
+
+          {(collapsible || tertiaryButtonLabel) && (
+            <footer className={`${blockClass}__footer`}>
+              {/* Disable the collapsible feature if an image is visible */}
+              {collapsible && !media && (
+                <Button
+                  className={`${blockClass}__toggle-btn`}
+                  kind="ghost"
+                  size="md"
+                  onClick={() => {
+                    setIsCollapsed((prevState) => !prevState);
+                  }}
+                >
+                  {isCollapsed ? expandButtonLabel : collapseButtonLabel}
+                </Button>
+              )}
+              {tertiaryButtonLabel && (
+                <Button
+                  className={`${blockClass}__close-btn`}
+                  size="md"
+                  onClick={onClick}
+                  kind="tertiary"
+                  renderIcon={Crossroads16}
+                >
+                  {tertiaryButtonLabel}
+                </Button>
+              )}
+            </footer>
+          )}
+        </div>
+        {media &&
+          (media.render ? (
+            <div className={`${blockClass}__media`}>{media.render()}</div>
+          ) : (
+            <SteppedAnimatedMedia
+              className={`${blockClass}__media`}
+              filePaths={media.filePaths}
+            />
+          ))}
+      </div>
+    );
+  }
+);
+
+// Return a placeholder if not released and not enabled by feature flag
+InlineTip = pkg.checkComponentEnabled(InlineTip, componentName);
+
+// The display name of the component, used by React. Note that displayName
+// is used in preference to relying on function.name.
+InlineTip.displayName = componentName;
+
+// The types and DocGen commentary for the component props,
+// in alphabetical order (for consistency).
+// See https://www.npmjs.com/package/prop-types#usage.
+InlineTip.propTypes = {
+  /**
+   * Provide the contents of the InlineTip.
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * Provide an optional class to be applied to the containing node.
+   */
+  className: PropTypes.string,
+  /**
+   * Tooltip text and aria label for the Close button icon.
+   */
+  closeIconDescription: PropTypes.string,
+  /**
+   * The label for the collapse button.
+   * This button is not visible if `media` is specified.
+   */
+  collapseButtonLabel: PropTypes.string,
+  /**
+   * If set to `true`, it will truncate the body text to
+   * one line and expose an expand/collapse button toggle.
+   *
+   * This feature is disabled if `media` is specified.
+   */
+  collapsible: PropTypes.bool,
+  /**
+   * Optional "call to action" ghost button or link that can appear
+   * directly below the content. This component comes with pre-styled
+   * elements available to use: `InlineTipLink` and `InlineTipButton`.
+   */
+  contentCTA: PropTypes.node,
```

Comment:
I think we typically call something with similar function something more like `action(s)` so maybe just `action` if that makes sense (see [`PageHeader`](https://v1-ibm-products.carbondesignsystem.com/?path=/story/ibm-products-components-page-header-pageheader--with-breadcrumbs&args=pageActions:1) and [`EmptyStates`](https://v1-ibm-products.carbondesignsystem.com/?path=/story/ibm-products-patterns-empty-state-emptystate--default))

Reply:
We have two props that are actions:
- `contentCTA` is a link or button using available, pre-styled components. This is part of the "collapsible" content and is not visible if collapsed.
- `tertiaryButtonLabel` is specifically a tertiary button with a "crossroads" icon. If the label is provided, the button is visible to the user regardless of the collapsed state.

I admit these are not as clear as I would like them. Any suggestions in clarifying these names would be appreciated.